apps/namadillo/src/App/Common/BugReport.tsx

```diff
@@ -0,0 +1,87 @@
+import { ActionButton, Panel } from "@namada/components";
+import { defaultAccountAtom } from "atoms/accounts";
+import {
+  isShieldedSyncCompleteAtom,
+  lastCompletedShieldedSyncAtom,
+} from "atoms/balance";
+import { chainStatusAtom } from "atoms/chain";
+import {
+  indexerCrawlersInfoAtom,
+  indexerHeartbeatAtom,
+  settingsAtom,
+} from "atoms/settings";
+import { useKeychainVersion } from "hooks/useKeychainVersion";
+import { useAtomValue } from "jotai";
+import { useRef } from "react";
+import { FaBug } from "react-icons/fa6";
+import { bugReportUrl } from "urls";
+import { version as sdkVersion } from "../../../../../packages/sdk/package.json";
+import { version as namadilloVersion } from "../../../package.json";
+
+export const BugReport = (): JSX.Element => {
+  const keychainVersion = useKeychainVersion();
+  const indexerHealth = useAtomValue(indexerHeartbeatAtom);
+  const crawlerInfo = useAtomValue(indexerCrawlersInfoAtom);
+  const chainStatus = useAtomValue(chainStatusAtom);
+  const currentDate = useRef(new Date().toString());
+  const account = useAtomValue(defaultAccountAtom);
+  const lastSyncInfo = useAtomValue(lastCompletedShieldedSyncAtom);
+  const isShieldedSyncComplete = useAtomValue(isShieldedSyncCompleteAtom);
+  const settings = useAtomValue(settingsAtom);
+  const lastShieldedSyncForAccount =
+    account.data?.address ? lastSyncInfo[account.data.address] : "?";
+
+  const yesOrNo = (bool: unknown): string => (!!bool ? "Yes" : "No");
+
+  const sep = "\n------------------";
+  const debugInfo = `## Navigator${sep}
+User Agent: ${navigator.userAgent}</li>
```

Comment:
This can be removed -> `</li>`

Noticed it in the bug report:
```
User Agent: Mozilla/5.0 (X11; Linux x86_64; rv:135.0) Gecko/20100101 Firefox/135.0</li>
```